package.json

```diff
@@ -35,12 +35,14 @@
   "files": [
     "lib"
   ],
-  "devDependencies": {},
+  "devDependencies": {
+    "mocha": "^5.2.0"
```

Comment:
Mocha 6 is out now, might be worth bumping this!

Reply:
Will look at this tonight, before I create a new release.

Reply:
Just update this. 👍

Reply:
Just upgraded this 👍